makedist

```diff
@@ -298,49 +303,66 @@ if $INCLUDEHELPER; then
 fi
 
 FUSESUBDIR=false
-CVMFSFUSE3URL="$CVMFSURL/`echo $CVMFSRPMNAME|sed 's/cvmfs-/cvmfs-fuse3-/'`"
+CVMFSFUSE3RPMNAME=`echo $CVMFSRPMNAME|sed 's/cvmfs-/cvmfs-fuse3-/'`
+CVMFSFUSE3URL="$CVMFSURL/$CVMFSFUSE3RPMNAME"
+
+USEFUSE3=false
 if $SING; then
+    USEFUSE3=true
+fi
+if $FUSE3; then
+    USEFUSE3=true
+fi
+if [ "$EL" -ge 9 ]; then
+    USEFUSE3=true
+fi
+
+if [ "$DISTRO" = suse ]; then
+    if $USEFUSE3; then
+        # hack to check if package is available in current dir
+        if [ ! -f "$HERE/$CVMFSFUSE3RPMNAME" ]; then
+            echo "No cvmfs-fuse3 for suse15 available." >&2
```

Comment:
Just use `suse` instead of `suse15` so it can still be used if other suse versions are supported in the future.